File: app/src/main/java/com/example/myapp/AboutFragment.java
package com.example.myapp;

import android.os.Bundle;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.Toast;


public class AboutFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        View view = inflater.inflate(R.layout.fragment_about, container, false);

        Button buttonDev = (Button) view.findViewById(R.id.button);

        // Inflate the layout for this fragment



        buttonDev.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast toast = Toast.makeText(v.getContext(), "Розроблена Бережанським Дмитром", Toast.LENGTH_LONG);
                toast.show();
            }
        });

        return view;
    }
}

File: app/src/main/java/com/example/myapp/AboutFragment.kt
package com.example.myapp

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.Toast
import androidx.fragment.app.Fragment

class AboutFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_about, container, false)
        val buttonDev = view.findViewById<View>(R.id.button) as Button

        // Inflate the layout for this fragment
        buttonDev.setOnClickListener { v ->
            val toast = Toast.makeText(v.context, "Розроблена Бережанським Дмитром", Toast.LENGTH_LONG)
            toast.show()
        }
        return view
    }
}